docs/diagrams/UnfavoriteSequenceDiagram.puml

```diff
@@ -0,0 +1,105 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":UnfavoriteCommandParser" as UnfavoriteCommandParser LOGIC_COLOR
+participant ":UnfavoriteCommand" as UnfavoriteCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":AddressBook" as AddressBook MODEL_COLOR
+participant ":UniquePersonList" as UniquePersonList MODEL_COLOR
+participant ":Person" as Person MODEL_COLOR
+end box
+
+[-> LogicManager : execute("unfav 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("unfav 1")
+activate AddressBookParser
+
+create UnfavoriteCommandParser
+AddressBookParser -> UnfavoriteCommandParser
+activate UnfavoriteCommandParser
+
+UnfavoriteCommandParser --> AddressBookParser
+deactivate UnfavoriteCommandParser
+
+AddressBookParser -> UnfavoriteCommandParser : parse("1")
+activate UnfavoriteCommandParser
+
+create UnfavoriteCommand
+UnfavoriteCommandParser -> UnfavoriteCommand
+activate UnfavoriteCommand
+
+UnfavoriteCommand --> UnfavoriteCommandParser :
+deactivate UnfavoriteCommand
+
+UnfavoriteCommandParser --> AddressBookParser
+deactivate UnfavoriteCommandParser
+
+AddressBookParser --> LogicManager :
+deactivate AddressBookParser
+
+LogicManager -> UnfavoriteCommand : execute()
+activate UnfavoriteCommand
+
+UnfavoriteCommand -> UnfavoriteCommand : execute(model)
```

Comment:
I think `exectute(model)` should be the previous one from LogicManager? Since it is being called from there. 

It is also too large, I think the diagram can be split up into smaller parts (before execute and after execute), and I believe some less relevant methods can be omitted such as self calls and CommandResult

Ref: https://nus-cs2103-ay2122s1.github.io/website/admin/tp-deliverables.html#deliverable-developer-guide-dg